docs/audits.md

```diff
@@ -642,6 +642,31 @@ In general, users should use for [Github Actions environment files]
         echo "$HOME/.local/my-bin" >> "$GITHUB_PATH"
     ```
 
+## `github-env`
+
+| Type     | Examples           | Introduced in | Works offline  | Enabled by default |
+|----------|--------------------|---------------|----------------|--------------------|
+| Workflow  | [github-env.yml]   | v0.6.0        | ✅             | ✅                 |
+
+[github-env.yml]: https://github.com/woodruffw/gha-hazmat/blob/main/.github/workflows/github-env.yml
+
+Detects dangerous usages of `$GITHUB_ENV` environment variable.
```

Comment:
```suggestion
Detects dangerous usages of the `GITHUB_ENV` environment variable.
```

Reply:
Done [here](https://github.com/woodruffw/zizmor/pull/192/commits/f222f8655f3f0d4a184982ef551822fbd9c07fea) with all suggestions for docs 🙂